using-mobile-extensions/mobile-core/identity/identity-event-reference.md

```diff
@@ -2,68 +2,248 @@
+| `updatedurl`            | String                    | Yes          | The update URL when `appendVisitorInfoForURL` (Android) / `appendToUrl` (iOS) public API is called |
+| `urlvariables`          | String                    | Yes          | The Visitor IDs as query parameters string when `getUrlVariables` API is called |
+
+#### Event data example
+
+When all identifiers, blob and location hint are populated
+
+```json
+{
+    "mid": "midExample",
+    "advertisingidentifier": "advertisingIdExample",
+    "pushidentifier": "pushIdExample",
+    "blob": "blobExample",
+    "locationhint": "locationHintExample",
+    "visitoridslist": [{
+        "id_type": "type1",
+        "id_origin": "origin1",
+        "id": "id1",
+        "authentication_state": 1
+    },
+    {
+        "id_type": "type2",
+        "id_origin": "origin2",
+        "id": "id2",
+        "authentication_state": 2
+    }]
+}
+```
+
+#### Event data example
+
+The result for the appendToUrl request
+
+```json
+{
+    "updatedurl": "https://example.com/path?query=parameter1&other=parameter2&adobe_mc=TS%3D1563845864296%7CMCMID%3Dmidexample%7CMCORGID%3Dorgidexample%40AdobeOrg%7CMCAID%3Daidexample&adobe_aa_vid=videxample"
+}
+```
+
+#### Event data example
+
+The result for the getUrlVariables request
+
+```json
+{
+    "urlvariables": "adobe_mc=TS%3D1563845864296%7CMCMID%3Dmidexample%7CMCORGID%3Dorgidexample%40AdobeOrg%7CMCAID%3Daidexample&adobe_aa_vid=videxample"
+}
+```
 
 {% hint style="warning" %}
-The Analytics Custom Visitor ID \(VID\) is no longer included in the data payload. If needed, it may be retrieved from the [Analytics extension](https://github.com/Adobe-Marketing-Cloud/aep-sdks-documentation/tree/174e9069bc1d3a521b59d52a066e9a7730f60ff5/using-mobile-extensions/adobe-analytics/analytics-api-reference/README.md#get-the-custom-visitor-identifier).
+
+The Analytics Custom Visitor ID \(VID\) is no longer included in the Identity event data payload. If needed, it may be retrieved from the [Analytics extension](/using-mobile-extensions/adobe-analytics/analytics-api-reference/README.md#get-the-custom-visitor-identifier).
+
 {% endhint %}
 
+### Configuration content request 
+
+The Identity Extension dispatches a new Configuration Content Request event when a privacy change is detected in the response received from the Experience Cloud ID Service. This event will be handled by the Core extension and the new privacy status will be broadcasted to other extensions.
+
+For more details about this event see the Mobile Core section.
+
+### Analytics content request
+
+The Identity Extension dispatches a new Analytics Content Request event when a push status preference change is detected, when the push identifier is initially set or when it is cleared from the SDK. This event will be handled by the Analytics extension and a new internal hit will be sent to the Adobe Analytics Server.
```

Comment:
- please add a comma between "initially" and "or" in the 1st sentence.
- Please reword the 2nd sentence so that it reads as follows:
"This event is handled by the Analytics extension, and a new internal hit is sent to the Adobe Analytics Server."